docs/guides/primitives.mdx

```diff
@@ -60,163 +46,35 @@ For hardware providers, implementing primitives natively means you can provide y
-
-```text
-a = SparsePauliOp("Z") # shape ()
-b = SparsePauliOp("IIIIZXYIZ") # shape ()
-c = SparsePauliOp.from_list(["XX", "XY", "IZ"]) # shape ()
-```
-
-The following lists of operators, while equivalent in terms of information contained, have different shapes:
-
-```text
-list1 = SparsePauliOp.from_list(["XX", "XY", "IZ"]) # shape ()
-list2 = [SparsePauliOp("XX"), SparsePauliOp("XY"), SparsePauliOp("IZ")] # shape (3, )
-```
-</Admonition>
+<span id="sampler"></span>
+## Sampler
 
-#### Sampler V1
+Sampler's core task is sampling the output register from execution of quantum circuits. The input circuits can be parametrized, as long as the parameter values are also provided as input to the primitive.
 
-<Admonition type="warning">
-[SamplerV1](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV1) has been deprecated in `qiskit-ibm-runtime` 0.23.
-Its support will be removed on 15 August 2024.
-[SamplerV2](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV2) is highly recommended instead.
-</Admonition>
+The input is one or more PUBs, in the format  (`<single circuit>`, `<optional one or more parameter values>`, `<optional shots>`), where there can be multiple `parameter values` items, and each item can be either an array or a single parameter, depending on the chosen circuit.
 
-* Takes two parameters: circuits and parameter values.
-* Each parameter can be a single value or a list.
-* The length of the parameters must match.
-* Elements from each are aggregated.
+The output is counts or per-shot measurements, as `PubResult` objects, without weights. The result class, however, has methods to return weighted samples, such as counts.
 
-Example:
+Input and output notes:
 
-```python
-sampler.run([circuit1, circuit2, ...],[observable1, observable2, ...],[param_values1, param_values2, ...] )
-```
-
-#### Sampler V2
-
-* Takes one parameter: PUBs in the format  (`<single circuit>`, `<optional one or more parameter values>`, `<optional shots>`), where there can be multiple `parameter values` items, and each item can be either an array or a single parameter, depending on the chosen circuit.
-* Elements from each are aggregated. For example, each array of parameter values in the PUB is applied to the PUB's circuit.
+* Elements from each PUB are aggregated. For example, each array of parameter values in the PUB is applied to the PUB's circuit.
 * Obeys program outputs. Typically this is a bit array but can also be an array of complex numbers (measurement level 1).
 * Returns raw data type. Data from each shot is returned (analogous to `memory=True` in the `backend.run` interface), and post-processing is done by using convenience methods.
 * Output data is grouped by output registers. You need the classical register name to get the results. By default, it is named `meas` if you use `measure_all()`. You can find the classical register name by running `<circuit_name>.cregs`. For example, `qc.cregs`.
```

Comment:
This list was originally written as a comparison between V1 and V2. It's a bit odd to have it here. In fact I don't think we need them anymore since the inputs/outputs [page](https://docs.quantum.ibm.com/guides/primitive-input-output) already covers them.